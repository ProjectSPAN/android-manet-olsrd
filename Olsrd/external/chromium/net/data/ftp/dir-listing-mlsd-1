type=dir;sizd=512;modify=20010414155043;UNIX.mode=0555;unique=6ag5b4e3ff; bin
type=dir;sizd=512;modify=20010414155237;UNIX.mode=0555;unique=6ag5b4e400; etc
type=dir;sizd=512;modify=20010705213550;UNIX.mode=0711;unique=6ag5b4e401; hidden
type=dir;sizd=512;modify=20050422001005;UNIX.mode=0755;unique=100ff03g6164000; j
type=dir;sizd=512;modify=20031021200128;UNIX.mode=0755;unique=6ag5b4e403; pub
