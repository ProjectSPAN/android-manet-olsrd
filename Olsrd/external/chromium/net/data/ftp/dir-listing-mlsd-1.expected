d
bin
-1
2001
4
14
15
50

d
etc
-1
2001
4
14
15
52

d
hidden
-1
2001
7
5
21
35

d
j
-1
2005
4
22
0
10

d
pub
-1
2003
10
21
20
1
